code/modules/recycling/conveyor2.dm

```diff
@@ -315,6 +315,13 @@
 					conveyors += C
 			return
 
+//CHOMPedit: Conveyor belts can be fast :)
+	if(istype(I, /obj/item/weapon/tool/wirecutters))
+		if(panel_open)
+			toggle_speed()
+			to_chat(user, "You cut the safety wire of the conveyor switch.")

```

Comment:
If I use a wirecutter on a sped up conveyor, does this mean I cut a second safety wire to slow it down again?

Reply:
fuck